visualization/app/codeCharta/ui/fileChooser/fileChooser.component.ts

```diff
@@ -38,7 +40,10 @@ export class FileChooserController {
 				}
 
 				reader.onload = event => {
-					content = (isCompressed ? zlib.unzipSync(Buffer.from(<string>event.target.result)) : event.target.result).toString()
+					const result: string = event.target.result.toString()
```

Comment:
```suggestion
					const result = event.target.result.toString()
```
I'd recommend to use type inference as much as possible. Especially if you call a function like `toString()` it's quite obvious, what the type is.

Reply:
Thanks :D !
Corrected in the next commit.